primitives/src/market.rs

```diff
@@ -79,6 +79,105 @@ impl<AI, BA, BN, M, A> Market<AI, BA, BN, M, A> {
     pub fn is_redeemable(&self) -> bool {
         matches!(self.resolution_mechanism(), ResolutionMechanism::RedeemTokens)
     }
+
```

Comment:
We should add documentation to these functions.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1281/commits/1a1768fd3ea64242c41e19a48408b930800d0cf3